docker/retag-lambda/cuda_versions.txt

```diff
@@ -1,8 +1,4 @@
 cu118
+cu117
 cu116
-cu113
-cu112
-cu111
-cu110
-cu102
-cu101
\ No newline at end of file
+cu115
```

Comment:
why are we back supporting cu115 and cu117? are there users that require those?

I guess the question is more like, why we started supporting cu116 and cu118 without supporting cu115 and cu117?

Reply:
This is due to the criteria we laid out in the support framework https://github.com/ray-project/ray/issues/34863

Generally we said that we'll support the CUDA versions that are also actively maintained by nvidia. This includes CUDA 11.5 and 11.7. I believe the main reason we didn't add them back then was because we already had a large number of GPU images and didn't want to inflate it further. I don't think it was a conscious decision - I think we had requests for 11.6 and later for 11.8, so we added them.

What we should have done is followed CUDA's lifecycle earlier and deprecated those images earlier. 

I'm also happy to leave those images out, but we'll have to find a good reasoning for this. I mainly care to have a plan going forward and a clear indication on which CUDA versions to add, keep, or remove, and following the CUDA image lifecycle seems easiest in terms of management.